pkgs/data/themes/openzone/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, icon-slicer, xcursorgen }:
+stdenv.mkDerivation rec {
+  pname = "openzone";
+  version = "1.2.9";
+
+  src = fetchFromGitHub {
+    owner = "ducakar";
+    repo = pname;
+    rev = "v${version}";
```

Comment:
I can't find a tag `v1.2.9` in the openzone repository, and `nix-build` also fails for me for that reason.

Reply:
Something has gone wrong with renaming the package. I will fix it in a second.